spec-insert/spec/_fixtures/expected_output/param_tables.md

```diff
@@ -17,15 +19,17 @@ api: search
 component: query_parameters
 include_global: true
 pretty: true
-columns: Type, Parameter, Description, Required, Default
+columns: Data Type, Parameter, Description, Required, Default
 -->
 ## Query parameters
-| Type    | Parameter                 | Description                                                                                                                        | Required | Default |
-|:--------|:--------------------------|:-----------------------------------------------------------------------------------------------------------------------------------|:---------|:--------|
-| Boolean | `analyze_wildcard`        | If true, wildcard and prefix queries are analyzed. This parameter can only be used when the q query string parameter is specified. | Required | `false` |
-| String  | `analyzer`                | Analyzer to use for the query string. This parameter can only be used when the q query string parameter is specified.              |          |         |
-| Boolean | `pretty`                  | Whether to pretty format the returned JSON response.                                                                               |          |         |
-| Boolean | `human` <br> _DEPRECATED_ | _(Deprecated since 3.0: Use the `format` parameter instead.)_ Whether to return human readable values for statistics.              |          | `true`  |
+The following table lists the available query parameters.
```

Comment:
Could you add a blank line between the section headers and the intro sentence (globally).

Reply:
Done!